### ISSUE TITLE
owl-carousel jQuery plugin is better or v-owl-carosel

### ISSUE THREAD

**Issue by imojiz** [93gaurav93/v-owl-carousel#15]
I want use owl-carousel jQuery plugin in my project. which is better for my project? owl-carousel jQuery plugin or v-owl-carosel? and why? which has more speed?

**Comment by 93gaurav93** [comment]
Are you using VueJS in your project?

**Comment by imojiz** [comment]
> Are you using VueJS in your project?

Yes. I using VueJS and Vuex

**Comment by mayur19** [comment]
@imojiz You can use jQuery plugin or v-owl-carousel both are having their plus points. If you choose jQuery plugin into VueJS then you'll need to perform some extra steps for integration (Installation is complicated for beginners. Great choice when you are using some themes which having jQuery plugin for carousel). Using Jquery plugin you'll have all the options to customize. If you choose v-owl-carousel then installation is straight forward for VueJS. But you'll have fewer options as compared to the jQuery plugin.

**Comment by imojiz** [comment]
> @imojiz You can use jQuery plugin or v-owl-carousel both are having their plus points. If you choose jQuery plugin into VueJS then you'll need to perform some extra steps for integration (Installation is complicated for beginners. Great choice when you are using some themes which having jQuery plugin for carousel). Using Jquery plugin you'll have all the options to customize. If you choose v-owl-carousel then installation is straight forward for VueJS. But you'll have fewer options as compared to the jQuery plugin. These is all my thoughts because I used jQuery plugin before but @93gaurav93 definitely guide well.

Has any difference between owl-carousel and v-owl carousel in speed? I want use that in my main page for 4 time in different sections.

**Comment by mayur19** [comment]
> > @imojiz You can use jQuery plugin or v-owl-carousel both are having their plus points. If you choose jQuery plugin into VueJS then you'll need to perform some extra steps for integration (Installation is complicated for beginners. Great choice when you are using some themes which having jQuery plugin for carousel). Using Jquery plugin you'll have all the options to customize. If you choose v-owl-carousel then installation is straight forward for VueJS. But you'll have fewer options as compared to the jQuery plugin. These is all my thoughts because I used jQuery plugin before but @93gaurav93 definitely guide well.
> 
> Has any difference between owl-carousel and v-owl carousel in speed? I want use that in my main page for 4 time in different sections.

Yes. There is difference between speed when it comes to rendering multiple times on the same page. You should use v-owl-carousel for better speed. Because it will take a single component. In the case of jQuery, you'll need to initiate/Install for each slider. So it will definitely take more time.
